### PR TITLE
mosh: Fix patch

### PR DESCRIPTION
**Pull request by ndowens**
assert(false); in patch causes the fish shell
to fail to start, essentially causing lock in
a mobile terminal

### CODE REVIEW

File: srcpkgs/mosh/patches/ignore-unknown-renditions.patch [hunk 1]
```diff
@@ -41,7 +41,7 @@ index a724afa6..93a31553 100644
    case 5: case 25: set_attribute(blink, value); break;
    case 7: case 27: set_attribute(inverse, value); break;
    case 8: case 28: set_attribute(invisible, value); break;
-+  default: assert(false);
++default: break;
```
Comment (Chocimier) [inline srcpkgs/mosh/patches/ignore-unknown-renditions.patch]: This is equal to not patching.

Comment (leahneukirchen) [reply srcpkgs/mosh/patches/ignore-unknown-renditions.patch]: But that patch is wrong, why would you crash mosh in this case?!

Comment (leahneukirchen) [reply srcpkgs/mosh/patches/ignore-unknown-renditions.patch]: Also it was reverted upstream in https://github.com/mobile-shell/mosh/commit/a40b67691aecfe9b31c3e8babf8e94d8d1a187a1

Comment (ndowens) [reply srcpkgs/mosh/patches/ignore-unknown-renditions.patch]: It's funny, looking at the patch it shows break, but one applied with same title inserts assert(false)

Comment (ndowens) [reply srcpkgs/mosh/patches/ignore-unknown-renditions.patch]: I removed the patch, and mosh+fish works

**Pull request merged**